pkg/p2p/peer/peer.go

```diff
@@ -226,7 +240,10 @@ func (p *Reader) ReadLoop() {
 
 	for {
 		// Refresh the read deadline
-		p.Conn.SetReadDeadline(time.Now().Add(readWriteTimeout))
+		err := p.Conn.SetReadDeadline(time.Now().Add(readWriteTimeout))
+		if err != nil {
+			l.WithError(err).Warnf("write timeout err: %v", err)
```

Comment:
In accordance with the style guide, please don't format the error into the `msg` field - having the `WithError` is enough.

Reply:
Done. 
That reminds that we'd need to release a coding guidelines doc that defines such rules/advises.